src/subresource-signing.js

```diff
@@ -39,8 +54,10 @@ var srs = {
         /**
          * Assume the contents is valid, and attach the script
          * to the document so it runs.
+         *
+         * @param contents
          */
-        attach_script: function srs_attach_script(contents) {
+        attach_script: function(contents) {
```

Comment:
The article @patricknelson posted seems to argue against his own point. The reason for using named functions is for easier debugging in stack traces:

> What it all boils down to is the fact that named function expressions is the only way to get a truly robust stack inspection.